kong/pdk/private/response/handler/grpc.lua

```diff
@@ -0,0 +1,157 @@
+local http_constants = require "kong.tools.http_constants"
```

Comment:
The grpc response handler may is the hardest thing to review. As the code style just changed for the code review.